### PR TITLE
Plato mods

### PR DESCRIPTION
**Pull request by clintonstimpson**
These modifications allow building Plato with moris

### CODE REVIEW

File: share/cmake/utilities/CMakeLists.txt
Comment (clintonstimpson) [inline share/cmake/utilities/CMakeLists.txt:1]: I suggest comparing this to changes to CMakeLists.txt in #1.

File: projects/HMR/src/CMakeLists.txt
Comment (kkmaute) [inline projects/HMR/src/CMakeLists.txt:1]: Merged with main branch

File: share/cmake/utilities/CMakeLists.txt
Comment (kkmaute) [inline share/cmake/utilities/CMakeLists.txt:1]: I am not sure which fix is better; I took the one from pr #1 but I am happy to use this one instead.

**Comment by clintonstimpson** [comment]
Thanks!

**Pull request closed without merging**